Source/HyprMXAdapter.swift

```diff
@@ -191,3 +201,18 @@ extension HyprMXAdapter: HyprMXInitializationDelegate {
         initializationCompletion = nil
     }
 }
+
+extension HyprConsentStatus: CustomStringConvertible {
+    public var description: String {
```

Comment:
I would avoid making this public, since it means we are modifying some HyprMX SDK type and making it public to all modules. Could lead to conflicts at some point.
Can we instead make it an internal extension, do not conform to `CustomStringConvertible`, and just call `log(.privacyUpdated(setting: "HyprConsentStatus", value: consentState.description))` instead?

Reply:
[done](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-hyprmx/pull/11/commits/60621b64aac9f962662e001c71ec61fa87e046d9)

Reply:
ty!